Extensions/PrimitiveDrawing/shapepainterruntimeobject.ts

```diff
@@ -237,6 +237,22 @@ namespace gdjs {
       );
     }
 
+    drawRegularPolygon(
+      centerX: float,
+      centerY: float,
+      radius: float,
+      sides: float,
+      rotation: float
+    ) {
+      this._renderer.drawRegularPolygon(
+        centerX,
+        centerY,
+        radius,
+        sides,
```

Comment:
You changed the order parameter in the action declaration, so you have to change it here as well (in both `shapepainterruntimeobject` and `shapepainterruntimeobject-pixi-renderer`)